text/0000-rust-crate-ownership.md

```diff
@@ -0,0 +1,158 @@
+
+An example of such messaging would be text like:
+
+> This crate is maintained by \[team\] as a part of an experiment around \[thingy\]. We encourage people to try to use this crate in their projects and provide feedback through \[method\], but do not guarantee long term maintenance.
+
+or, for experiments that are not intended to be used at all:
+
+> This crate is maintained by \[team\] and is an internal experiment. We do not guarantee stability or long term maintenance, please use at your own risk.
+
+Ideally, experimental crates that are published for feedback purposes will have a document to link to that lists out the purpose, rough duration, and processes of the experiment.
+
+### Deprecated
+“Deprecated” crates should contain the following text near the top of the readme/documentation:
+
+> This crate is deprecated and not intended to be used.
+
+### Placeholder
+
+“Placeholder” crates should contain the following text in their published readme/documentation:
+
+> This crate is a functionally empty crate that exists to reserve the crate name of \[tool\]. It should not be used. 
+
+In general it is better to have an empty placeholder crate published instead of reserving the crate via yanking, so that there is a readme that helps
+
+
+### Expatriated
+
+It's unclear if any action should be taken on these beyond removing any semblance of officialness (including rust-lang/foo team owners). We currently have only one such crate (`rand`).
+
+These should by and large not be considered to be "team managed" crates; this category is in this RFC for completeness to be able to talk about expatriation as an end state.
+
+## Transitions and new crates
+
+Teams should feel free to create new crates in any of these categories; however "Intentional Artifact" crates must be accompanied with an RFC. As we move towards having team charters, this can transition to being a charter change (which may require an RFC or use its own process). Teams should notify core@rust-lang.org when they've created such crates so that the core team may track these crates and ensure this policy is applied.
+
+From time to time a team's plan for a crate may change: experiments may conclude, crates may need to be deprecated, or the team may decide to release something for wider usage.
+
+In general, teams should notify core@rust-lang.org when such a transition is being made.
+
+Any transition _away_ from "Intentional Artifact" requires an RFC.
+
+Any transition to "Intentional Artifact" should ideally be accompanied by an RFC, and an update to the team charter if there is one.
+
+Expatriation should basically _never_ occur anymore, but it also requires an RFC and core team approval in case it is really necessary. If a team wishes to stop working on a crate, they should deprecate it and encourage the community to fork it or build their own thing.
```

Comment:
What's the reason for this policy? I'm sure there is a good reason, but it would be helpful for readers if it were spelled out in the text.

Reply:
See https://github.com/rust-lang/rfcs/pull/3119#discussion_r625885524.

Reply:
Thanks! Sorry I missed that. It would definitely be good for it to get added to the text.